docs/content/dagster-cloud/developing-testing/dagster-cloud-cli.mdx

```diff
@@ -5,3 +5,163 @@ noindex: True
 ---
 
 # Using the Dagster Cloud CLI
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+The `dagster-cloud` CLI is a command-line toolkit designed to work with Dagster Cloud.
+
+In this guide, we'll cover how to install and configure the `dagster-cloud` CLI, get help, and use some helpful environment variables and CLI options.
+
+---
+
+## Installing the CLI
+
+The Dagster Cloud Agent library is available in PyPi. To install, run:
+
+```shell
+pip install dagster-cloud
```

Comment:
This could be `pip install dagster-cloud-cli` (docs are slightly out of date - both work, but dagster-cloud-cli just has the CLI). The thing you type in the command line is still `dagster-cloud`.